tests/test_msdb.py

```diff
@@ -33,9 +35,9 @@ def test_get_spoiler_from_id(self):
 
     def test_get_all_users(self):
         mock_users = [
-            User(('Alice', 0, 0)),
-            User(('Bob', 1, 1)),
-            User(('Carol', 2, 2)),
+            User(('Alice', 0, 0, '{}')),
```

Comment:
Right now the `User` and `Spoiler` objects are instantiated using a row_tuple since their only function is to be extracted from the DB and passed around the main application - this makes instantiation a bit weird.

I've been meaning to fix this with an overloaded constructor but it just hasn't been on my priority list.

Can we deal with this in another PR?